src/dataEntryApp/sagas/programEncounterSaga.js

```diff
@@ -8,18 +8,22 @@ import {
   setProgramEncounterForm,
   setProgramEncounter,
   saveProgramEncounterComplete,
-  setValidationResults
+  setValidationResults,
+  setCancelProgramEncounterForm
 } from "../reducers/programEncounterReducer";
 import api from "../api";
 import {
   selectFormMappingForSubjectType,
-  selectFormMappingByEncounterTypeUuid
+  selectFormMappingByEncounterTypeUuid,
+  selectCancelProgramEncounterFormMapping
 } from "./programEncounterSelector";
 import { mapForm } from "../../common/adapters";
 import { ProgramEncounter, ProgramEnrolment, ObservationsHolder, Concept } from "avni-models";
 import { ModelGeneral as General, EncounterType } from "avni-models";
-import { getSubjectProfile } from "../reducers/subjectDashboardReducer";
+import { getSubjectProfile, setSubjectProfile } from "../reducers/subjectDashboardReducer";
 import { mapProgramEncounter } from "common/subjectModelMapper";
+import { setLoad } from "../reducers/loadReducer";
```

Comment:
Why do we need a separate reducer for load status? Generally, the loading status is part of each specific page and so belongs to the page specific reducer.